src/pages/collection/Collection.jsx

```diff
@@ -6,43 +6,60 @@ import { peopleOutline } from "ionicons/icons";
 import useUserContext from "../../hooks/useUserContext";
 import AddButton from "../../components/button/AddButton";
 import EditButton from "../../components/button/EditButton";
+import UnfollowButton from "../../components/button/UnfollowButton";
+import FollowButton from "../../components/button/FollowButton";
 import CategoryChip from "../../components/chip/CategoryChip";
 import ImageGrid from "../../components/gallery/ImageGrid";
 import HomeToolbar from "../../components/toolbar/HomeToolbar";
 import { getCollectionByCollectionId } from "../../services/collections";
 import "./Collection.scss";
 import CollectionItems from "../../components/collection-items/CollectionItems";
 import Text from "../../components/text/Text";
+import { followByCollectionId, unfollowByCollectionId } from "../../services/followers";
+import FollowersList from "./FollowersList";
+import { useLocation } from "react-router";
 
 const Collection = (props) => {
+  const location = useLocation();
   const history = useHistory();
   // const { title = "Test Collection", ownerName = "Test", ownerUsername = "test", description = "Test Collection Description..." } = props;
   const { collectionId } = useParams();
   const [title, setTitle] = useState("");
   const [description, setDescription] = useState("");
-  const [ownerUserId, setOwnerUserId] = useState(null);
-  const [ownerName, setOwnerName] = useState("User's Name");
+  const [ownerId, setOwnerId] = useState(null);
   const [ownerUsername, setOwnerUsername] = useState("Username");
   const [loading, setLoading] = useState(false);
   const [categoryId, setCategoryId] = useState(null);
   const [categoryName, setCategoryName] = useState(null);
+  const [followed, setFollowed] = useState(false);
+  const [followersCount, setFollowersCount] = useState(0);
 
   const { currentUserId } = useUserContext();
 
+  const isCollectionOwner = Number(currentUserId) === Number(ownerId);
```

Comment:
Does this update if either variable changes? I don't think there's any code that updates this